libraries/Gamebuino-Meta/src/utility/Sound/Sound.h

```diff
@@ -77,6 +113,10 @@ class Sound {
 	};
 	
 	int8_t play(Sound_Handler* handler, bool loop = false);
+
+	void fx(const Sound_FX & fx);
```

Comment:
Auto-detect length via templates?

Reply:
The sound FX structure has a hidden flag that says if the fx should continue or not, like this
```
Gamebuino_Meta::Sound_FX sfx_test[8] =
{
	{ Gamebuino_Meta::Sound_FX_Wave::SQUARE_CONTINUE, 19968,0,8192,0,1000 },
	{ Gamebuino_Meta::Sound_FX_Wave::SQUARE, 19968,-5,4096,0,10000 },
};
```

Reply:
Ah yes, that will do it, too